api_server.py

```
@@ -561,4 +565,5 @@ def add_security_headers(response):
 """
 
 if __name__ == "__main__":
-    app.run(host="0.0.0.0", port=int(os.environ.get("PORT", 5000)))
+    # Binding to 0.0.0.0 is intentional for containerization
+    app.run(host="0.0.0.0", port=int(os.environ.get("PORT", 5000)))  # nosec B104
```

Ruff (0.15.5)

[error] Possible binding to all interfaces

(S104)